autoload/leaderf/python/leaderf/lineExpl.py

```diff
@@ -63,6 +63,7 @@ def _defineMaps(self):
     def _accept(self, file, mode, *args, **kwargs):
         lfCmd("norm! m'")
         super(LineExplManager, self)._accept(file, mode, *args, **kwargs)
+        lfCmd('silent! norm! zO') #fix 473
```

Comment:
If issue #492 is the root cause, I think the fix has already fixed 473.

Reply:
On second thought, I don't think it is the root cause. There is no reason it would be opened uncollapsed, I think.  Try to press `:348` where 348 is a collapsed line. It would jump there but it would stay collapsed.

Reply:
so you mean the issue still exists? can you provide a minimum vimrc to reproduce it?